src/libcommon/utility/types.h

```diff
@@ -332,19 +334,42 @@ enum class AppStateKey {
     LogUploadState,
     LogUploadPercent,
     LogUploadToken,
+    AppUid,
     Unknown  //!\ keep in last position (For tests) /!\\ Only for initialization purpose
 };
+
 constexpr int64_t SELF_RESTARTE_DISABLE_VALUE = -1;
 constexpr int64_t SELF_RESTARTER_NO_CRASH_DETECTED = 0;
 
 enum class LogUploadState { None, Archiving, Uploading, Success, Failed, CancelRequested, Canceled };
-
 enum class UpdateState { Error, None, Checking, Downloading, Ready, ManualOnly, Skipped };
 
+enum class UpdateStateV2 { UpToDate, Available, Downloading, Ready, Error };
+enum class DistributionChannel { Prod, Next, Beta, Internal, Unknown };
+enum class Platform { MacOS, Windows, LinuxAMD, LinuxARM, Unknown };
+
+struct VersionInfo {
+        std::string tag;                      // Version number. ex: 3.6.4
```

Comment:
```suggestion
        std::string tag;                      // Version number, for example: 3.6.4
```

Reply:
Does the name `tag` reflects the fact that it coincides with the `git tag` of the release branch.

Reply:
As a matter of fact, it does. But no, it reflects the key retrieved from the GetAppVersion request,